src/curated-corpus/pages/CustomSectionsPage/CustomSectionsPage.tsx

```diff
@@ -0,0 +1,488 @@
+import React, { useEffect, useState } from 'react';
+import { DateTime } from 'luxon';
+import {
+  Section,
+  SectionStatus,
+  useGetScheduledSurfacesForUserQuery,
+  useGetSectionsWithSectionItemsLazyQuery,
+} from '../../../api/generatedTypes';
+import { DropdownOption } from '../../helpers/definitions';
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+  Paper,
+  Chip,
+  TextField,
+  MenuItem,
+} from '@mui/material';
+import { HandleApiResponse } from '../../../_shared/components';
+
+interface GroupedSections {
+  scheduled: Section[];
+  live: Section[];
+  expired: Section[];
+}
+
+export const CustomSectionsPage: React.FC = (): JSX.Element => {
```

Comment:
this is good! 
can we align the `start`, `end`, `status` columns in all 3 section groups?